src/platform/user/authentication/utilities.js

```diff
@@ -15,6 +16,11 @@ export const ssoKeepAliveEndpoint = () => {
   return `https://${envPrefix}eauth.va.gov/keepalive`;
 };
 
+function inboundSSOe() {
+  // TODO: check to see if inbound ssoe is enabled
```

Comment:
unfortunately this is gonna require accessing the redux store where all the feature flags live right now, which has 2 options I'm not crazy about:

1. import the redux store and [access it directly](https://redux.js.org/api/store#getstate). right now the store is initialized in `src/platform/startup/index.js` and isn't exported, so I'm having a hard time seeing a clean way to do that w/ minimal changes
2. add another argument with the flag and pass in the feature flag from a connected component that's calling login. that's probably easier right now, but `sessionTypeUrl is already getting nasty with the amount of arguments it's taking.

if route 2 seems to be the best option, then maybe changing `sessionTypeUrl` accept an object as an argument and destructure it to be more legible, eg

```js
sessionTypeUrl({ type = '', version = 'v0', application, to, queryParams = {}, inboundSSOe}){
 // ....
}
```
I'm not crazy about either, but with the redux store as the source of truth, not sure if there's a way around the messiness

Reply:
@hinzed1127 I think I found another way around this.  In the AutoSSO container I've added a condition to `setForceAuth` if inbound is disabled, that way we don't need to run that check in the utilities function.  Could you take another look?